aten/src/ATen/core/Vitals.cpp

```diff
@@ -34,11 +38,31 @@ bool torchVitalEnabled() {
   return enabled;
 }
 
+std::string APIVitals::readVitals() {
+  if (!torchVitalEnabled()) {
+    return "";
+  }
+
+  std::stringstream buf;
+  for (auto x : name_map_) {
+    buf << x.second.toString();
```

Comment:
Why we are only returning the "second" values here, but not "first"? In the test below, it looks like both the name and the value should be returned, say 'CUDA.used\t\t true'.